main.cpp

```diff
@@ -0,0 +1,33 @@
+// This is a program that calculates stock information
+// Created by NAOMI BROWN on 1/31/16.
+
+#include <iostream>
+#include <iomanip>
+
+// Constants
+const int SHARES_BOUGHT = 1000;
+const int SHARES_SOLD = 1000;
+const double COMMISSION_RATE = .02;
+const double PURCHASE_PRICE = 45.50;
+const double SALE_PRICE = 56.90;
+
+// Variables
+double totalPurchasePrice = SHARES_BOUGHT*PURCHASE_PRICE;
+double totalBuyCommission = totalPurchasePrice * COMMISSION_RATE;
+double totalSalePrice = SHARES_SOLD*SALE_PRICE;
+double totalSaleCommission = totalSalePrice*COMMISSION_RATE;
+double totalProfit = (totalSalePrice-totalSaleCommission)-(totalPurchasePrice+totalBuyCommission);
+
+int main() {
+    std::cout<< "Total amount paid for stock was $"
+        <<std::setprecision(7)<<std::showpoint<< totalPurchasePrice<<std::endl;
+    std::cout<< "Total amount of commission paid to broker when stock was bought was $"
+        <<std::setprecision(5)<<std::showpoint<< totalBuyCommission<< std::endl;
```

Comment:
You should also declare variables as close to usage as possible. In the old days we were told to declare them at the top of the file but this is no longer the case. Now we should do something like this. With variables being somewhat "one and done". Used once and then garbage collected.

Example:
```
// Purchase Info
double totalPurchasePrice = SHARES_BOUGHT*PURCHASE_PRICE;
double totalBuyCommission = totalPurchasePrice * COMMISSION_RATE;
std::cout<< "Total amount paid for stock was $"
     <<std::setprecision(7)<<std::showpoint<< totalPurchasePrice<<std::endl;
std::cout<< "Total amount of commission paid to broker when stock was bought was $"
    <<std::setprecision(5)<<std::showpoint<< totalBuyCommission<< std::endl;

// Sales Info
double totalSalePrice = SHARES_SOLD*SALE_PRICE;
double totalSaleCommission = totalSalePrice*COMMISSION_RATE;
std::cout<<"Total amount of stock sold was $"
     <<std::setprecision(7)<<std::showpoint<< totalSalePrice<< std::endl;
std::cout<<"Total amount of commission paid to broker when stock was sold was $"
     <<std::setprecision(6)<<std::showpoint<< totalSaleCommission<< std::endl;
```